packages/bot-skeleton/src/scratch/blocks/Binary/After Purchase/after_purchase.js

```diff
@@ -61,7 +61,9 @@ Blockly.Blocks.after_purchase = {
             event.type === Blockly.Events.BLOCK_CHANGE ||
             (event.type === Blockly.Events.BLOCK_DRAG && !event.isStart)
         ) {
-            removeExtraInput(this);
+            if (this.isCollapsed()) {
+                modifyMainBlockOnCollapse(this);
```

Comment:
```suggestion
                appendCollapsedMainBlocksFields(this);
```

Can we use this name instead?